bundles/org.openhab.binding.souliss/README.md

```diff
@@ -192,13 +187,14 @@ Thing t11 10-0 "Albero di Natale" [node=10,slot=0]
 Thing t11 11-0 "Birra"@"Soppalco" [node=11,slot=0]
 Thing t52 11-1 "Birra - Temp 1"@"Soppalco" [node=11,slot=1]
 Thing t52 11-3 "Birra - Temp 2"@"Soppalco" [node=11,slot=3]
+
+Thing t13 0-12 "Valvola Riscaldamento Zona Giorno"@"Riscaldamento" [node=0,slot=12]
```

Comment:
I see that other existing strings here are in Italian. Nevertheless, as they should be in English, please make sure that at least newly added strings are in English.

Reply:
> I have added a few comments, but also have one question: It seems this PR doesn't fix anything per se, but rather removes channel `stateOnOff` as an alternative to `stateOpenClose` (supporting the same underlying logic). Is this correct? If this channel was actually working, this is a breaking change, and should be mentioned here: https://github.com/openhab/openhab-distro/blob/221e59eab7137d4b80aa539f37cdb104939cab13/distributions/openhab/src/main/resources/bin/update.lst#L77

@jlaur on the 3.3 section ? It will be released on 3.4

Reply:
No, new section 3.4 should be added after that line. :)